pkg/apis/openebs.io/v1alpha1/cstor_volume_claim.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2019 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
+
+// CStorVolumeClaim describes a cstor volume claim resource created as
+// custom resource
+type CStorVolumeClaim struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              CStorVolumeClaimSpec   `json:"spec"`
+	Status            CStorVolumeClaimStatus `json:"status"`
+}
+
+// CStorVolumeClaimSpec is the spec for a CStorVolumeClaim resource
+type CStorVolumeClaimSpec struct {
+	Capacity       string                  `json:"capacity"`
+	Publish        map[string]string       `json:"publish,omitempty"`
```

Comment:
Publish should be a part of CStorVolumeClaim struct instead of CStorVolumeClaimSpec according to the doc.

Reply:
`CStorVolumeClaimSpec`  is a part of `CStorVolumeClaim` as  Spec